src/modules/universe/RadixNodeConnection.ts

```diff
@@ -306,6 +306,14 @@ The atom may or may not have been accepted by the node.
             })
     }
 
+    public async getUniverse() {
```

Comment:
Any new public methods should have a tsdoc comment(also appreciated if you add it to any old methods that don't have it if you have a chance)

Reply:
Noted. Although this method was not used anymore, so I removed it.